README.md

```diff
@@ -11,9 +11,9 @@
 [![Slack](http://slack.layer5.io/badge.svg)](http://slack.layer5.io)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3564/badge)](https://bestpractices.coreinfrastructure.org/projects/3564)
 
-<p align="center"><i>If you’re using Meshery or if you like the project, please <a href="https://github.com/layer5io/meshery/stargazers">★</a> star this repository to show your support! 🤩</i></p>
+<p align="center"><i>If you’re using Service Mesh Performance or if you like the project, please <a href="https://github.com/layer5io/service-mesh-performance/stargazers">★</a> star this repository to show your support! 🤩</i></p>
```

Comment:
@leecalcote The `service-mesh-performance-specification` repository needs to be renamed to `service-mesh-performance` for this to work.

Reply:
@chandrashritii, should I make changes? I trimmed out `specification` from some places under the assumption that the repository's name will change. 😐

Reply:
We can build on this in the future if needed but for the moment, we're to good to go here. The repo's name has been changed @JubayerJoy

Reply:
Good work 👍  @JubayerJoy

Reply:
@chandrashritii thank you 😃